index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title>HTML & CSS Training</title>
+    <meta name="description" content="training" />
+    <meta name="author" content="Wolox" />
+    <link rel="stylesheet" type="text/css" href="application.css" />
+  </head>
+  <body>
+    <div class="book">
+      <div>
+        <img
+          class="book-cover"
+          src="./images/book-cover.png"
+          alt="book-cover"
+        />
```

Comment:
```suggestion
        <img class="book-cover" src="./images/book-cover.png" alt="book-cover" />
```

No need to write it in multiple lines 😄

Reply:
Fixed in #3